app/presenters/claim/advocate_claim_presenter.rb

```diff
@@ -54,4 +54,8 @@ def mandatory_case_details?
   def requires_interim_claim_info?
     claim.agfs_reform?
   end
+
+  def any_prosecution_evidence
```

Comment:
is this method still required? does not look like it is being called anywhere now

Reply:
Doesn't look like it only place it's being called is in the tests.

Reply:
Have removed method and tests 👍